### ISSUE TITLE
Problem in French explanation of Window.devicePixelRatio

### ISSUE THREAD

**Issue by leodarrigade** [mdn/translated-content#12593]
### MDN URL

https://developer.mozilla.org/fr/docs/Web/API/Window/devicePixelRatio

### What specific section or headline is this issue about?

Main text

### What information was incorrect, unhelpful, or incomplete?

La propriété en lecture seule Window.devicePixelRatio retourne le rapport entre la taille (verticale) d'un pixel physique sur le périphérique d'affichage et la taille d'un pixel indépendant du matériel (abrégé en anglais "dips").

### What did you expect to see?

La propriété en lecture seule Window.devicePixelRatio retourne le rapport entre **la taille d'un pixel indépendant du matériel (abrégé en anglais "dips") et la taille (verticale) d'un pixel physique sur le périphérique d'affichage .**

### Do you have any supporting links, references, or citations?

From the English version of the same webpage ([here](https://developer.mozilla.org/en-US/docs/Web/API/Window/devicePixelRatio)):
"
This value could also be interpreted as the ratio of pixel sizes: the size of one CSS pixel to the size of one physical pixel. In simpler terms, this tells the browser how many of the screen's actual pixels should be used to draw a single CSS pixel. 
"

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `fr/web/api/window/devicepixelratio`
* MDN URL: https://developer.mozilla.org/fr/docs/Web/API/Window/devicePixelRatio
* GitHub URL: https://github.com/mdn/translated-content/blob/main/files/fr/web/api/window/devicepixelratio/index.md
* Last commit: https://github.com/mdn/translated-content/commit/040a7573de416de6d0defcdb706a19924bdd44d6
* Document last modified: 2022-09-21T09:43:04.000Z

</details>

**Comment by SphinxKnight** [comment]
Thanks a lot @leodarrigade. The page would benefit from an overall update. I'm going to tackle this.
